src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdClientSettings.Edit.cshtml

```diff
@@ -54,7 +54,7 @@
 
 <div class="mb-3" asp-validation-class-for="Scopes">
     <label asp-for="Scopes" class="form-label">@T["Scopes"]</label>
-    <input asp-for="Scopes" class="form-control" />
+    <input asp-for="Scopes" class="form-control"autocomplete="off" />
```

Comment:
Referring to the screenshot below, there are two places where it is recommended to add directly to the form element

Also, the added attribute omits the space ：）
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/3fa83c2c-bc6d-4c47-afd2-e4dbdc585e75)

Reply:
> there are two places where it is recommended to add directly to the form element

Where are they?

> Also, the added attribute lost a space ：）

Really? which browser? coz we using this attribute in several places

Reply:
Ignore the screenshot above, it appears to be out of date

I'm using Edge, the other place I'm referring to is `Client Secret`, the latest version of the code I looked at seems to have added the autocomplete property.

Reply:
OK , the above screenshot is indeed from the latest version of the source code, the Scopes element has indeed not been adjusted, but the `Client Secret` has been adjusted, and it doesn't seem to be working as expected, it's still autofilling (note that the textboxes in the red area on the left contain a light blue background).

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/f52769c0-361e-4095-9882-ef05fa91c75c)

Reply:
Do you mean the current auto-filling? I don't think so. Please the tested browser

Reply:
The Client Secret needs to be not auto-complete too indeed. I see the same issue with Chrome.

Reply:
The auto-complete is already Off, if you tested recently I will check in Chrome

Reply:
If you have credentials saved for the host, then it'll auto-fill them here still:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/cb3df692-5fe1-48db-8aba-3d179b098902)

I have admin/Password1! saved for OC login. Check it after saving something like this.

Reply:
This is a strange problem, I applied the changes in the code, but the problem still exists, if you open and save in an anonymous window, the autocompletion really doesn't work, but it still autofills in the default browser!

Reply:
I didn't see the Client Secret in my screen, is there any configuration

Reply:
> I didn't see the Client Secret in my screen, is there any configuration

Yes， you have to check  `Use code response type` , Refer to the screenshot above